e2e/test/e2e-tx-broken-by-gas-limit.test.ts

```diff
@@ -0,0 +1,54 @@
+import { expect } from "chai";
+import { keccak256 } from "ethers";
+import { Block, Transaction, TransactionReceipt } from "web3-types";
+
+import { ALICE, Account, BOB, randomAccounts } from "./helpers/account";
+import { isStratus } from "./helpers/network";
+import {
+    CHAIN_ID,
+    CHAIN_ID_DEC,
+    HASH_EMPTY_TRANSACTIONS,
+    HASH_EMPTY_UNCLES,
+    ONE,
+    TEST_BALANCE,
+    TEST_TRANSFER,
+    ZERO,
+    fromHexTimestamp,
+    send,
+    sendGetBalance,
+    sendRawTransaction,
+    sendReset,
+} from "./helpers/rpc";
+
+describe("GAS_LIMIT", () => {
+    var _tx: Transaction;
```

Comment:
Most of these values like `_txSentTimestamp` are not being used.
Please remove it.
They are declared here when we want to pass data from one step to another.
If this is not the case, they are not necessary here.